src/utils.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 const Config = require('../config.json')
 const Messages = require('./messages')
+const { in_progress } = require('../ranking-info/ranking.json')
```

Comment:
Do not import the `ranking.json` file in `utils.js`. Since later the ranking.json can change to a Database i want to keep the imports of that file restricted.

Please change the `removeAlreadyChallengedPlayers` to receive a 3rd argument, the `in_progress`argument and [when calling `getPlayersThatCanBeChallenged`](https://github.com/Xotl/Smash-League/pull/52/files#diff-39a666c7bcedf10cf714e8b239a65805L121) you can pass the whole `Ranking` object instead of `Ranking.ranking`, which is basically the `ranking.json` file so you can get `Ranking.in_progress`.

Reply:
![image](https://user-images.githubusercontent.com/1013633/61235735-d447fb00-a6fb-11e9-87a1-9545e42d065f.png)